Makefile

```diff
@@ -88,6 +88,7 @@ HSE_VALUE       ?= 8000000
 FEATURES        =
 
 OFFICIAL_TARGETS  = ALIENFLIGHTF3 ALIENFLIGHTF4 ANYFCF7 BETAFLIGHTF3 BLUEJAYF4 CC3D FURYF4 NAZE REVO SIRINFPV SPARKY SPRACINGF3 SPRACINGF3EVO SPRACINGF3NEO SPRACINGF4EVO STM32F3DISCOVERY
+SKIP_TARGETS     := ALIENWHOOP
```

Comment:
I think this should at least print a message indicating that a target has been skipped, otherwise it will be the source for endless frustration.